# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/components/windowwatcher/nsWindowWatcher.cpp
# Commit: 6955906262c0
# Full Hash: 6955906262c0920b0f9f28dcf3177e9ce5e0dffc
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2020-04-07 03:53:09
# Regressor Bug: 1616353
# File Overlap Count: 1
# Description:
#   Bug 1616353 - Part 8: Remove forceInitialBrowserNonRemote, r=Gijs
#   
#   This method is no longer necessary, as frontend will always make the correct
#   decisions for the remoteness of the intial content browser by reading
#   openWindowInfo.
# ==============================================================================

diff -r a7700472807a -r 6955906262c0 toolkit/components/windowwatcher/nsWindowWatcher.cpp
--- a/toolkit/components/windowwatcher/nsWindowWatcher.cpp	Mon Apr 06 14:30:19 2020 +0000
+++ b/toolkit/components/windowwatcher/nsWindowWatcher.cpp	Mon Apr 06 14:30:22 2020 +0000
@@ -949,16 +949,6 @@
       }
 
       if (newChrome) {
-        nsCOMPtr<nsIAppWindow> appWin = do_GetInterface(newChrome);
-        if (appWin) {
-          nsCOMPtr<nsIXULBrowserWindow> xulBrowserWin;
-          appWin->GetXULBrowserWindow(getter_AddRefs(xulBrowserWin));
-          if (xulBrowserWin) {
-            nsPIDOMWindowOuter* openerWindow =
-                aForceNoOpener ? nullptr : parentWindow.get();
-            xulBrowserWin->ForceInitialBrowserNonRemote(openerWindow);
-          }
-        }
         /* It might be a chrome AppWindow, in which case it won't have
             an nsIDOMWindow (primary content shell). But in that case, it'll
             be able to hand over an nsIDocShellTreeItem directly. */